reflex_cli/user_input.py

```diff
@@ -133,6 +133,14 @@ def ask_to_overwrite(file_path):
     def verify_upgrade_interest(self):
         """Prompts user whether or not they want to upgrade rule."""
         if self.interactive:
-            verify = self.get_input("Upgrade? (y/n):")
-            return verify.lower() == "y"
-        return False
+            upgrade_interest = [
+                {
+                    "type": "confirm",
+                    "message": "Upgrade?",
+                    "name": "upgrade_interest",
+                    "default": False,
+                }
+            ]
+            verify = prompt(upgrade_interest)
+            return verify["upgrade_interest"]
+        return True
```

Comment:
I see this was returning False before, is this to fix the issue you were seeing with the `--all` flag?

Reply:
Yep